test/dataSources.spec.js

```diff
@@ -142,6 +152,33 @@ describe('data_sources', () => {
         }));
   });
 
+  describe('getTables', () => {
```

Comment:
Tests don't run (and never have for this file), but these should at least inform how it should be & what the responses look like.